tests/avocado/test_ovmf_with_240_vcpus.py

```diff
@@ -0,0 +1,44 @@
+import os
+import shutil
+import sys
+
+from avocado import Test
+from base import VM
+
+
+class TestOvmfVcpus(Test):
+    """
+    :avocado: tags=ovmf
+    """
+
+    def setUp(self):
+        ovmf_secboot = os.path.join(self.teststmpdir, 'OVMF_CODE.secboot.fd')
+        if not os.path.exists(ovmf_secboot):
+            shutil.copy('/usr/share/edk2/ovmf/OVMF_CODE.secboot.fd',
+                        self.teststmpdir)
+        ovmf_vars = os.path.join(self.teststmpdir, 'OVMF_VARS.fd')
+        if not os.path.exists(ovmf_vars):
+            shutil.copy('/usr/share/edk2/ovmf/OVMF_VARS.fd',
+                        self.teststmpdir)
+
+        args = ['-smp', '240']
+        args.extend(['-drive', 'file=%s' % ovmf_secboot])
+        args.extend(['-drive', 'file=%s' % ovmf_vars])
+
+        self.vm = VM(args)
+
+    def test_run_vm(self):
+        """
+        According to the BZ1447027, the issue is: Guest cannot boot with
+        240 or above vcpus when using ovmf.
+
+        :avocado: tags=BZ1447027
```

Comment:
To the upstream developers, way more important the Red Hat Bugzilla ID is a reference to a commit that fixed the Bug. I mean, adding a RHBZ reference is welcome, but if you can find the commit ID with a simple search, you'll add considerable value to your test and changelog.